integration

```diff
@@ -90,6 +90,7 @@
   "andrewjswan/SwatchTime",
   "andrzejchm/blebox_shutterbox_tilt",
   "andvikt/mega_hacs",
+  "andymcloid/bokat_se_hass",
```

Comment:
As this is only relevant for users in a specific country, the released `hacs.json` file needs a country key.
<https://hacs.xyz/docs/publish/include>